### ISSUE TITLE
Consider hiding the breadcrumb at the top level and changing its postion

### ISSUE THREAD

**Issue by at055612** [google/docsy#857]
When you are at the top level of a section the breadcrumb shows a single link like so

![image](https://user-images.githubusercontent.com/22818309/150518960-f29511e2-f599-4ae7-9c58-a2dffbe862cc.png)![image](https://user-images.githubusercontent.com/22818309/150519886-7557728d-9efd-4189-98d0-7f97120b414b.png)

which looks a little odd as it is not obvious that is is a breadcrumb.  With >1 item it is fine

Thus consider changing `breadcrumb.html` to hide the nav for a single item.
```html
{{ $isSingle := true -}}
{{ with .Parent -}}
  {{ $isSingle = .IsHome -}}
{{ end -}}
{{ if not $isSingle }}
<nav aria-label="breadcrumb" class="td-breadcrumbs
    {{- if $isSingle }} td-breadcrumbs__single {{- end }}">
  <ol class="breadcrumb">
    {{- template "breadcrumbnav" (dict "p1" . "p2" .) }}
  </ol>
</nav>
{{ end }}
...
```
For me this then looks like:

![image](https://user-images.githubusercontent.com/22818309/150521653-251783c7-4cdd-4977-9aaa-1a45e5f290b2.png)

Also, when you have the archived banner visible, the breadcrumbs appear below it which looks a little odd.

![image](https://user-images.githubusercontent.com/22818309/150520596-e57cdbcb-2319-4681-a33c-2fed212e3646.png)

I think it looks better if the breadcrumbs always remain at the top like so

![image](https://user-images.githubusercontent.com/22818309/150520777-972eb056-bad8-4e91-b0f6-1da5a3bc2a99.png)

i.e. in `layouts/docs/baseof.html`
```html
...
          <main class="col-12 col-md-9 col-xl-8 pl-md-5" role="main">
            {{ if not .Site.Params.ui.breadcrumb_disable }}{{ partial "breadcrumb.html" . }}{{ end }}
            {{ partial "version-banner.html" . }}
            {{ block "main" . }}{{ end }}
          </main>
...
```

Appreciate this is all subjective.

**Comment by chalin** [comment]
You can hide single-element breadcrumbs using custom styling as mentioned in #786, that is:

```css
.td-breadcrumbs__single { display: none !important; }
```

FYI, There's already an issue open to ensure that that get's documented (#810).

**Comment by chalin** [comment]
As for the relative order of the breadcrumbs & banner, that's ... subjective, as you put it :). I prefer the current design in that respect.

**Comment by at055612** [comment]
Thanks for the `.td-breadcrumbs__single` tip.  Understand you position about the banner/breadcrumb order.  Happy to close.

**Comment by chalin** [comment]
@at055612, I agree with your request here, that most projects I've worked on prefer for the single breadcrumb in index pages to be hidden. Here's the change:

- #2160
